README.md

```diff
@@ -8,36 +8,64 @@ GitHub Page: https://github.com/MarcoFazioRandom/Virtual-Joystick-Godot
 
 Godot Engine: https://godotengine.org
 
-### PREVIEWS:
+## PREVIEWS:
 
 <img src="previews/preview1.png" width="400">
 
-Easy to setup:  
+Easy to setup:
+
 <img src="previews/preview2.png" width="200">
 
-Easy to use:  
-<img src="previews/preview3.png" width="500">
```

Comment:
Image is great in Asset Library, but unhelpful to get started once imported into your own project.